### PR TITLE
remove random src folder full of json files

### PR DESCRIPTION
**Pull request by JillieBeanSim**
we somehow have a random src folder in the next branch that is all json files from the localization that looks like it's from running prettier in this commit https://github.com/zowe/vscode-extension-for-zowe/pull/1509/commits/7b96ce2e1af2874d047298739af536e0bb9cfee8

image

## Types of changes

What types of changes does your code introduce to Zowe Explorer?
_Put an `x` in the boxes that apply_

- [x] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Updates to Documentation or Tests (if none of the other choices apply)

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This checklist will be used as reference for both the contributor and the reviewer_

- [x] I have read the [CONTRIBUTOR GUIDANCE](https://github.com/zowe/vscode-extension-for-zowe/wiki/Best-Practices:-Contributor-Guidance) wiki
- [x] PR title follows [Conventional Commits Guidelines](https://www.conventionalcommits.org/en/v1.0.0-beta.2/)
- [x] PR Description is included
- [x] gif or screenshot is included if visual changes are made
- [x] `yarn workspace vscode-extension-for-zowe vscode:prepublish` has been executed
- [ ] All checks have passed (DCO, Jenkins and Code Coverage)
- [ ] I have added unit test and it is passing
- [ ] I have added integration test and it is passing
- [ ] There is coverage for the code that I have added
- [ ] I have tested it manually and there are no regressions found
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any PR dependencies have been merged and published (if appropriate)

## Reminder

After a PR is merged into the `master` branch, create a PR from `master` to the `next` branch resolving any conflicts.

## Further comments

If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by JillieBeanSim** [comment]
we may want to check that all 1.18.1 items are brought in as well since we noticed that the @zowe/cli dependency that was added to the ZE API package.json was missing

**Comment by JillieBeanSim** [comment]
how do we fix an audit with a critical severity if no patch? https://github.com/advisories/GHSA-8p5q-j9m2-g8wr

**Comment by lauren-li** [comment]
> resolving ssh2 npm audit seems to break everything, removing that resolution until we figure this out. This is a dependency of @zowe/cli, do they have a version with the fix that we can update to?

I have created PR #1521, which updates the Zowe CLI version to `7.0.0-next.202109281609`. As the ssh2 audit issues were resolved in Zowe CLI version `7.0.0-next.202109141650`, PR #1521 should take care of this for Zowe Explorer's `next` branch. (Thanks @JillieBeanSim for your research on this!)

File: packages/zowe-explorer/src/Profiles.ts [hunk 1]
```diff
@@ -894,7 +894,7 @@ export class Profiles extends ProfilesCache {
                     const doNotSaveButton = localize("promptCredentials.doNotSave.button", "Do Not Save");
                     const infoMsg = localize(
                         "promptCredentials.saveCredentials.infoMessage",
-                        "Save entered credentials for future use with profile: {0}? Saving credentials will update the local yaml file.",
+                        "Save entered credentials for future use with profile: {0}?\nSaving credentials will update the local yaml file.",
```
Comment (lauren-li) [inline packages/zowe-explorer/src/Profiles.ts:897]: The `\n` here doesn't seem to create a newline for me in the pop-up message. I do see that this is in `master` branch, so I think it's okay to go into `next`, but I was curious if others can see its effect.

**Review by lauren-li — APPROVED** [review]
This looks good to me! I do not see any regressions as a result of this PR. Per @crawr and @VitGottwald's update, the lodash audit issue might be [revoked](https://github.com/lodash/lodash/issues/5261) in the near future (and Zowe Explorer does not use the `template` function of lodash, so we are not affected anyway).

The ssh2 audit issue can be handled in a separate PR (such as #1521).

Thanks @JillieBeanSim for cleaning up the folders (and taking care of some dependency issues)!

**Pull request merged**